### PR TITLE
Feature/improve unit coverage

### PR DESCRIPTION
**Pull request by wildone**
Add additional Unit Tests.

### CODE REVIEW

File: aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentDetailsUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    PageManager pageManager;
+
+    @Before
+    public void before() {
+        initMocks(this);
+    }
+
+    @Test
+    public void testClass() {
+        ComponentDetailsUtil test = new ComponentDetailsUtil();
```
Comment (leachuk) [inline aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentDetailsUtilTest.java]: This is redundant. You would get build/compilation errors if this class didn't exist

File: aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentsUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    @Before
+    public void before() {
+        initMocks(this);
+        PowerMockito.mockStatic(IOUtils.class);
+    }
+
+    @Test
+    public void testClass() {
+        ComponentsUtil test = new ComponentsUtil();
```
Comment (leachuk) [inline aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentsUtilTest.java]: Same as above, this is redundant.

File: aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentDetailsUtilTest.java (hunk 1 above)
Comment (ArpitVarshney18) [reply aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentDetailsUtilTest.java]: Removed now.

File: aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentsUtilTest.java (hunk 2 above)
Comment (ArpitVarshney18) [reply aemdesign-aem-services/src/test/java/design/aem/utils/components/ComponentsUtilTest.java]: removed

**Pull request closed without merging**